force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -230,16 +325,23 @@ public with sharing class ClassesObjectsMethods {
         // title
         // author
         // pages
+        public String title;
+        public String author;
+        public Integer pages;
     
         // Book constructor to initialize the title and author
-        public Book() {
-            // Initialize the title and author
+        public Book( String title, String author, Integer pages) {
+        // Initialize the title and author
+        this.title = title;
+        this.author = author;
+        this.pages = pages;
+
         }
     
         // Method that returns the details of the book
         public String getBookDetails() {
             // return A string containing the details of the book in the format "Title: <title>, Author: <author>"
-            return null; // Replace null with the variable you used to store the result
+            return 'Title: + <title>, + Author: +<author>, + Pages: +<pages>'; // Replace null with the variable you used to store the result
```

Comment:
The <> should be removed from around your variable names and you need to make sure you use single quotes around the Strings. Also, you should be using the instance value of title, author, and pages, which can be done using the 'this' keyword. For example, `this.title`